<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChapterEditor</class>
 <widget class="QWidget" name="ChapterEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="10">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>章节设置</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>下一章</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="6">
    <widget class="QLineEdit" name="lineNext"/>
   </item>
   <item row="0" column="8">
    <widget class="QPushButton" name="win_save">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="lineCurrent"/>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="Label">
     <property name="text">
      <string>本章</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="btnRemove">
     <property name="text">
      <string>删除本章</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
